src/utils.js

```diff
@@ -199,3 +199,12 @@ export function humanDate(since: string | number): string {
     second: "2-digit",
   }) + " UTC";
 }
+
+export function buildAttachmentUrl(record: RecordData, capabilities: Capabilities): ?string {
+  if (!record.attachment || typeof capabilities.attachments !== "object") {
+    return;
+  }
+  const {base_url} = capabilities.attachments;
+  const {location} = record.attachment;
+  return location.startsWith("http") ? location : base_url + location;
```

Comment:
What about `return location.startsWith("base_url") ? location : base_url + location;`?
Can we do something clever so that we make sure that `location` starts with a / and `base_url` doesn't end with one?